docs/consume-packages/install-use-packages-nuget-cli.md

```diff
@@ -20,6 +20,30 @@ For most commands, the NuGet CLI tool uses the current directory, unless you spe
 
 For a complete list of commands and their arguments, see the [NuGet CLI reference](../reference/nuget-exe-cli-reference.md).
 
+## Support policy
+
+We are fully committed to supporting the most recent version of NuGet.exe.
+This means you can rely on us for bug fixes, updates, and enhancements exclusive to the version currently under development.
+
+NuGet.exe follows the [Microsoft Modern Lifecycle Policy](https://aka.ms/lifecycle).
+
+Here are steps you can take to leverage the NuGet Support Policy effectively:
```

Comment:
Nobody leverages a support policy per-say. It just is.

Reply:
This line is directly from the blog post. I'll rewrite it.